### PR TITLE
ADR-811: Malted Barley Screen

### PR DESCRIPTION
**Pull request by sini-george**

### CODE REVIEW

File: test/forms/spiritsQuestions/MaltedBarleyUsedFormProviderSpec.scala [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import play.api.data.FormError
+import scala.collection.immutable.ArraySeq
+
+class MaltedBarleyUsedFormProviderSpec extends BigDecimalFieldBehaviours {
+
+  val form = new MaltedBarleyUsedFormProvider()()
+
+  ".value" - {
+
+    val fieldName = "maltedBarleyUsed-input"
```
Comment (JemmaEdwards) [inline test/forms/spiritsQuestions/MaltedBarleyUsedFormProviderSpec.scala:29]: I think the format here should be malted-barley-used-input

File: test/controllers/spiritsQuestions/MaltedBarleyUsedControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          .overrides(
+            bind[QuarterlySpiritsQuestionsNavigator].toInstance(new FakeQuarterlySpiritQuestionsNavigator(onwardRoute)),
+            bind[CacheConnector].toInstance(mockCacheConnector)
+          )
+          .build()
+
+      running(application) {
+        val request =
+          FakeRequest(POST, maltedBarleyUsedRoute)
+            .withFormUrlEncodedBody(("maltedBarleyUsed-input", validAnswer.toString))
```
Comment (JemmaEdwards) [inline test/controllers/spiritsQuestions/MaltedBarleyUsedControllerSpec.scala:104]: I think the format here should be malted-barley-used-input

File: app/views/spiritsQuestions/MaltedBarleyUsedView.scala.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+@layout(pageTitle = title(form, messages("maltedBarleyUsed.title"))) {
+
+    @formHelper(action = controllers.spiritsQuestions.routes.MaltedBarleyUsedController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @sectionHeading(
+            id = "maltedBarleyUsed-section",
```
Comment (JemmaEdwards) [inline app/views/spiritsQuestions/MaltedBarleyUsedView.scala.html:40]: I think the format here should be malted-barley-used-section and I should change my code too

File: app/views/spiritsQuestions/MaltedBarleyUsedView.scala.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            text = messages("section.alcoholDutyQuarterlySpiritsReturn"),
+        )
+
+        @govukInput(
+            InputViewModel(
+                field = form("maltedBarleyUsed-input"),
```
Comment (JemmaEdwards) [inline app/views/spiritsQuestions/MaltedBarleyUsedView.scala.html:46]: I think the format here should be malted-barley-used-input

**Review by JemmaEdwards — COMMENTED** [review]
couple of input values I noticed nothing major!

**Review by msatta — APPROVED** [review]

**Review by msatta — APPROVED** [review]

**Pull request merged**